debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerSnapshotChangeEventSource.java

```diff
@@ -94,15 +93,20 @@ protected void connectionCreated(SnapshotContext snapshotContext) throws Excepti
     @Override
     protected void lockTablesForSchemaSnapshot(ChangeEventSourceContext sourceContext, SnapshotContext snapshotContext) throws SQLException, InterruptedException {
         if (connectorConfig.getSnapshotLockingMode() == SnapshotLockingMode.NONE) {
+            // No table locks acquired. Read committed isolation level is used to avoid taking shared locks on rows.
+            jdbcConnection.connection().setTransactionIsolation(Connection.TRANSACTION_READ_COMMITTED);
+            LOGGER.info("Schema locking was disabled in connector configuration");
+        } else if (connectorConfig.getSnapshotLockingMode() == SnapshotLockingMode.CONSISTENT) {
```

Comment:
Minor nit: can you move the code after `}` to the next line?

Reply:
done